api/v2/views/email.py

```diff
@@ -13,11 +13,10 @@
 from api import permissions
 from api.v2.exceptions import failure_response
 
-from core.email import email_admin, resource_request_email
+from core.email import lookupEmail, resource_request_email, support_email, email_admin, request_data
```

Comment:
@lenards lookupEmail is wrapping `iplantauth.protocol.ldap.lookupEmail`. It's correct but could have a better name..